lib/views/shopping_list.erb

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
   <body>
-    <h1>Shopping List</h1>
+    <h1><%= @list.name %></h1>
     <ul>
-      <% @list_items.each do |item| %>
+      <% @list.list_items.each do |item| %>
```

Comment:
:cool:

we talked about how the view may now know entirely too much about the `List` implementation, but we aren't going to stress about it yet.

i feel like cleaner patterns will emerge as views get more complex.